### PR TITLE
Update Scala 2.12 & 2.13 (for ltsv4s 1.0.4)

### PR DESCRIPTION
**Pull request by kijuky**
# Summary 

Update Scala 2.12 & 2.13

Because [ltsv4s that is dependency library update for scala 2.12 & 2.13](https://github.com/seratch/ltsv4s/pull/5).

# Details

| library | old | new | remarks |
|----|----|----|----|
| scala | 2.11.5, 2.10.4 | 2.13.10, 2.12.17 | |
| sbt | 0.13.6 | 1.8.2 | |
| ltsv4s | 1.0.2 | 1.0.4 | |
| slf4j | 1.7.7 | 1.7.36 | |
| mockito-all | 1.9.5 | 1.10.19 | |
| scalatest | 2.2.1 | 3.2.5 | |
| scalatest mockito | | 3.2.5.0 | |
| scoverage | 0.99.7.1 | 2.0.6 | |
| sbt-scalariform | 1.3.0 | (deleted) | |
| coveralls | 0.99.0 | 1.3.6 | |

### CODE REVIEW

**Comment by lloydmeta** [comment]
Hey, thanks for this. I'll need to fix up a bunch of CI related things first before merging this, but it's on my todo.

**Comment by kijuky** [comment]
>  I'll need to fix up a bunch of CI related things first before merging this

OK. I created a pull request #7 to resolve #6. I'll merge #7 into this pull request later.

**Comment by kijuky** [comment]
I merged #7 with this commit b80a87f

- add java 11
- add java 17
- add scala 2.12
- add scala 2.13

- drop java 8
- drop scala 2.11

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,7 +7,8 @@ jobs:
     strategy:
       matrix:
         java_version: [
-          "adopt@1.8.0-292"
```
Comment (lloydmeta) [inline .github/workflows/test.yml:10]: Can we add this one back? I _think_ the removal of this is somewhat preventing the GH action from being run for this PR ?

**Review by lloydmeta — APPROVED** [review]
Overall LGTM; just one comment re: GH actions

File: .github/workflows/test.yml (hunk 1 above)
Comment (kijuky) [reply .github/workflows/test.yml:10]: > Can we add this one back?

java 1.8 is no longer supported, so I removed this. 

>  I think the removal of this is somewhat preventing the GH action from being run for this PR ?

If I change the GHAction it should be reflected in the PR as well. 
FYI, GH actions are working in my forked repo. https://github.com/kijuky/ltsv-logger/actions/runs/4151580580

Comment (lloydmeta) [reply .github/workflows/test.yml:10]: Hmm, ok odd. Let's go through with this and see what happens.

**Comment by lloydmeta** [comment]
@kijuky お待たせしました。I just pushed 0.1.0 to maven central with your changes, thanks so much.

お疲れ様です。

**Comment by kijuky** [comment]
Thank you for your work! It helped me a lot.

**Pull request merged**